packages/server/customer-os-api/repository/dashboard_repository.go

```diff
@@ -473,3 +474,91 @@ func (r *dashboardRepository) GetDashboardNewCustomersData(ctx context.Context,
 
 	return results, nil
 }
+
+func (r *dashboardRepository) GetDashboardCustomerMapData(ctx context.Context, tenant string) ([]map[string]interface{}, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "DashboardRepository.GetDashboardCustomerMapData")
+	defer span.Finish()
+	tracing.SetDefaultNeo4jRepositorySpanTags(ctx, span)
+
+	session := utils.NewNeo4jReadSession(ctx, *r.driver)
+	defer session.Close(ctx)
+
+	dbRecords, err := session.ExecuteRead(ctx, func(tx neo4j.ManagedTransaction) (any, error) {
+
+		queryResult, err := tx.Run(ctx, fmt.Sprintf(
+			`
+					MATCH (o:Organization)-[:HAS_CONTRACT]->(c:Contract)-[:ACTIVE_RENEWAL]->(op:Opportunity)
```

Comment:
please add link with tenant, otherwise archived contracts will appear in the list

Reply:
done